test/functional/treesitter/utils_spec.lua

```diff
@@ -30,4 +30,20 @@ describe('treesitter utils', function()
     eq(true, exec_lua('return vim.treesitter.is_ancestor(ancestor, child)'))
     eq(false, exec_lua('return vim.treesitter.is_ancestor(child, ancestor)'))
   end)
+
+  it('can convert a ts range to a vim range', function()
+    insert([[
```

Comment:
This needs to be gated behind a 
```lua
    if pending_c_parser(pending) then return end
```
since the `functionaltest-lua` action doesn't have a parser installed (for now).